doc/ADMIN.md

```diff
@@ -0,0 +1,4 @@
+* Email required for administation (uses "magic links" instead of admin passwords)
+
+* You should manually configure the app after installation.
```

Comment:
```suggestion
```

This doesnt bring any useful information

Reply:
Indeed.